js/src/Map.js

```diff
@@ -107,6 +108,7 @@ export class LeafletMapModel extends widgets.DOMWidgetModel {
       this.set('south', bounds.south);
       this.set('east', bounds.east);
       this.set('west', bounds.west);
+      this.set('window_url', window.location.href);
```

Comment:
You can put this line where the map is initialized?

Reply:
You mean in the `initialize` method of the view?

Reply:
It would be even better in the initialize method of the model, then it is called only once.

Reply:
Thanks.